deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -81,6 +82,18 @@ public BigDecimal getCirculatingSupply() {
     return lastMetric.getCirculatingSupply();
   }
 
+  /**
+   * Retrieves total Meeds supply, reading the value of the ERC20.totalSupply() Meeds contract
+   *
+   * @return {@link BigDecimal} for most recent computed total supply
+   *           value
+   */
+  public BigDecimal getTotalSupply() {
+    MeedTokenMetric lastMetric = getLastMetric();
+    BigDecimal totalSupply = lastMetric.getTotalSupply();
+    return totalSupply.setScale(8, RoundingMode.CEILING);
```

Comment:
why is this needed ?

Reply:
because we need 8 decimals format

Reply:
it wasn't mentioned explicitly here neither for other endpoints, so please keep it simple by including all decimals